samples/dashboards/demos/mathmodifier-formula/demo.html

```diff
@@ -0,0 +1,17 @@
+<script src="https://code.highcharts.com/dashboards.js"></script>
+<script src="https://code.highcharts.com/dashboards-math.js"></script>
```

Comment:
This is fine by me: 
`<script src="https://code.highcharts.com/dashboards/modules/math-modifier.js"></script>`

It is consistent with other packages like Stock and Maps:
`https://code.highcharts.com/stock/modules/exporting.js`

> How about a plugins folder? modules naming has become confusing with modern JavaScript, because the modules are not ES modules.

Yes, that's a good point. On the flip side it will not be consistent with Highcharts, Stock and Maps then. Also it is debatable whether it is a plugin - these modules are actually part of the product, while a plugin typically is either third party or at least not part of the product. Add-ons has the same problem.